categories/meson.build

```
@@ -0,0 +1,22 @@
+sizes = ['16', '24', '32', '48', '64', '128']
+
+links = [
+    # The first argument is the original icon name,
+    # the second is the link
+    ['applications-internet', 'preferences-desktop-online-accounts'],
+    ['multimedia-volume-control', 'preferences-desktop-sound'],
+    ['preferences-desktop-locale', 'config-language'], # gnome language selector
+    ['preferences-system-power', 'gnome-power-manager']
+]
+
+foreach link : links
+    foreach size: sizes
+        if (fs.exists(link[1] + '.svg'))
```

```suggestion
        if (fs.exists(link[1] + '.svg'))
```
This looks wrong to me, is anything installed?